### PR TITLE
feature/ak-52: Added 'Lambda Admin Priveleges' plugin and test cases

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

**Review by dipsubha06 — APPROVED** [review]

File: exports.js [hunk 1]
```diff
@@ -246,6 +246,7 @@ module.exports = {
         'ssmAgentLatestVersion'         : require(__dirname + '/plugins/aws/ssm/ssmAgentLatestVersion.js'),
         'ssmAgentAutoUpdateEnabled'     : require(__dirname + '/plugins/aws/ssm/ssmAgentAutoUpdateEnabled'),
 
+        'lambdaAdminPriveleges'         : require(__dirname + '/plugins/aws/lambda/lambdaAdminPriveleges.js'),
```
Comment (giorod3) [inline exports.js:249]: privilege is spelled wrong here.

File: plugins/aws/lambda/lambdaAdminPriveleges.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            if (!listFunctions.data.length) {
+                helpers.addResult(results, 0, 'No Lambda functions found', region);
+                return rcb();
+            }
+
+            async.each(listFunctions.data, function(lambdaFunc, fcb){
+                if (!lambdaFunc.FunctionArn) return fcb();
+
+                var resource = lambdaFunc.FunctionArn;
+                var adminPriveleged;
```
Comment (giorod3) [inline plugins/aws/lambda/lambdaAdminPriveleges.js:43]: misspelling here as well

File: plugins/aws/lambda/lambdaAdminPriveleges.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                                adminPriveleged = true;
+                                break;
+                            }
+                        }
+                    }
+                }
+
+                if (!adminPriveleged) {
+                    helpers.addResult(results, 0,
+                        'Function does not have admin priveleges', region, resource);
```
Comment (giorod3) [inline plugins/aws/lambda/lambdaAdminPriveleges.js:143]:
```suggestion
                        'Function does not have admin privileges', region, resource);
```

File: plugins/aws/lambda/lambdaAdminPriveleges.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                } else {
+                    helpers.addResult(results, 2,
+                        'Function has admin priveleges', region, resource);
```
Comment (giorod3) [inline plugins/aws/lambda/lambdaAdminPriveleges.js:146]:
```suggestion
                        'Function has admin privileges', region, resource);
```

**Review by giorod3 — CHANGES_REQUESTED** [review]

**Review by giorod3 — APPROVED** [review]

**Pull request merged**